requirements.txt

```diff
@@ -26,3 +26,4 @@ bump2version>=1.0.1
 # Earlier versions had different type annotations
 pyzmq>=22
 devcluster>=1,<2
+fastapi
```

Comment:
there're fastapi and pydantic imports in the generated files, so I suspect `harness/setup.py` needs that as a dependency.